README.md

```diff
@@ -15,6 +15,19 @@ composer create-project drupal/starshot-project --repository='{"type":"vcs","url
 ```
 This one command will install Starshot and open it in a web browser for you to play with. You'll get all the modules and themes listed below, pre-configured.
 
+### DDEV installation
+If you don't have PHP and Composer installed on your host machine, you can try in [DDEV](https://ddev.readthedocs.io):
+```
+mkdir starshot && cd starshot
+ddev config --project-type=drupal --docroot=web
+ddev start
+ddev composer create drupal/starshot-project --repository='{"type":"vcs","url":"https://github.com/phenaproxima/starshot-prototype"}' --stability=dev
+```
+After the download completes, you probably get an error since `drupal quick-start` doesn't work in DDEV. Stop the server with Ctrl-C, and create login link:
```

Comment:
You know, I have an idea for a hack here.

If you look at `composer.json`, you'll see that `post-create-project-cmd` only starts the server if the `$CI` environment variable is undefined. Is there a way for us to pass that variable to the `ddev composer create ...` command, thus bypassing the automatic server startup, and removing the need for this awkward workaround?

Reply:
That's a great idea! Let me know if you have anything for me to experiment with. Or, like I wrote elsewhere, you really ought to give DDEV a test-spin :)

Reply:
You know more about DDEV than I do, so you might be in the better position to experiment. :)

That said, I think I'm okay merging this PR as-is, once penyaskito confirms that it works. Making the DDEV experience easier is a worthy thing to do, but we can do it in other PRs. Getting _some_ instructions in front of DDEV users is beneficial immediately and we don't need to let perfect be the enemy of good.

Reply:
For reference, here are some DDEV environment variables: https://ddev.readthedocs.io/en/stable/users/extend/custom-commands/#environment-variables-provided

So in theory we could remove the workaround if we changed `composer.json` to say this:

```
test -n \"$CI\" || test -n \"$DDEV_TLD\" || composer drupal:run-server
```

Reply:
And hey...I bet we could remove the `ddev config` line too if we shipped a `.ddev/config.yml` with this project. I'd be totally open to that.

So in theory we could get it all the way down to:
```
composer create-project --no-install ...
cd starshot-project
ddev composer install
ddev composer drupal:install
ddev start
```

And I bet we could script it too.

Reply:
That could probably work, I am ready to try that. I am not sure how to do local Composer development ... So maybe you can update composer.json with that suggestion? Then I'll give it a spin, after it has propagated to packagist.org ...

Reply:
Should we create a spin off issue for this, and commit the README changes?

Reply:
I created #49.

Reply:
Yes, shipping a .ddev config would be great!

Reply:
Though, again -- the point of DDEV is that PHP and Composer is not installed on the host machine. So the first command `composer create-project --no-install ...` is a no go.

Reply:
_Ah yes_...forgot. Well, let's see what we can do.

I think we've taken _this_ PR as far as we need to; once the instructions are confirmed to work, we can keep improving DDEV stuff in #49.

Reply:
Perfect, thanks!